metrics.md

```diff
@@ -0,0 +1,47 @@
+ 
+#### Gluster prometheus end point schema: 
+
+#### Volume Metrics : 
+
+|Gluster Promethus schema| example| codes |
+|--------|-----------|---|
+|gluster_peer_status |	gluster_peer_status{nodeid="xxxxx"} 1 |	1 - connected, 0 - disconnected	
```

Comment:
shall i change nodeid to peerid for all the types or only for gluster_peer_status

Reply:
lets have uniform label peerid or nodeid whatever it may be. having different labels for same value creates confusion.

Reply:
Yes agree. Lets make it peerid in all the places.
Will have to send a PR to make the required changes in already merged PRs.

Reply:
@aravindavk @shtripat @vredara I am wondering if it would be helpful to have cluster-id as well in the metrics. There could be multiple clusters in the deployment and it would be useful to know that information. What do you guys think?